src/System.IO.Ports/src/Resources/Strings.resx

```diff
@@ -169,4 +166,7 @@
   <data name="PlatformNotSupported_IOPorts" xml:space="preserve">
     <value>System.IO.Ports is currently only supported on Windows.</value>
   </data>
+  <data name="PlatformNotSupported_SerialPort_GetPortNames" xml:space="preserve">
+    <value>Enumeration of serial port names is not supported on the current OS.</value>
```

Comment:
@danmosemsft is the casing "Platform" consistent with the typical usage in messages?